minetest/minetest
# Set color of scrollbar/dropdown button symbols and checkmarks to white

Add compact, short information about your PR for easier understanding:

- Goal of the PR

I have a contrast vision issue and have difficulties seeing the symbols on scrollbar and dropdown buttons.

This patch increases the color contrast of the symbols on scrollbar and dropdown buttons.

- How does the PR work?

Before this patch, symbols on buttons for scrollbars and dropdowns were rendered black on dark background.

This patch sets the default rendering color for the symbols to white.

- Does it resolve any reported issue?

Fixes https://github.com/minetest/minetest/issues/13909

- Does this relate to a goal in [the roadmap](https://github.com/minetest/minetest/blob/master/doc/direction.md)?

Yes, general correctness (black on dark gray is incorrect according to many accessability standards) and/or UI improvements.

## How to test

1. Start Minetest without this patch.
2. Open the settings dialog.
3. Verify that the symbols on scrollbar and dropdown buttons are black on dark background, like in this screenshot:

![minetest-580-dev-settings-fail-1](https://github.com/minetest/minetest/assets/60905/7ff3c0f7-1448-4fa4-bf4d-c08b1f8b3c57)

4. Start Minetest with this patch.
5. Open the settings dialog.
6. Verify that the symbols on scrollbar and dropdown buttons are white on dark background, like in this screenshot:

![minetest-580-white-symbols-on-scrollbar-and-dropdown-buttons](https://github.com/minetest/minetest/assets/60905/0a65895a-ba6a-42f6-9c27-1eb4e6ae3108)

7. Check with different values of GUI scaling (I tested 1.0, 2.0, 3.0, 4.0) if the checkmarks are all white. Restart Minetest to get the checkbox to resize.
8. Verify that the checkmark in the mod loading dialog for a world still looks white (it should be a bit brighter than before even).

**Edit:** If the color in my screenshot appears off-white it is probably because I use redshift to adjust the display's color temperature.
